apps/codeforafrica/src/lib/data/pagify/stories.js

```diff
@@ -0,0 +1,16 @@
+import { getPost } from "@/codeforafrica/lib/data/utils/posts";
+
+async function stories(api, context) {
```

Comment:
Similar to opportunities, is this for single story or stories?

Reply:
This is to handle single story and opportunity

Reply:
Then the function/file should be named `story` and not `stories`.